square/square.go

```diff
@@ -0,0 +1,22 @@
+package square
+
+// Square ...
+type Square struct {
+	StartPoint Point
+	Side       uint
+}
+
+// End ...
+func (s Square) End() (uint, uint) {
+	return s.Side, s.Side
```

Comment:
`End()` should return a `Point` structure with the coordinates of the `End` Point (see picture in the lecture).

Reply:
Done https://github.com/VSKut/go-temp/pull/2/commits/460a5dbb243a35d2772d15d7a7190fa02d76f234